From 3d56fb135965658732aeebc8be47df861fea76f5 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 22 Apr 2021 15:22:33 +0000
Subject: [PATCH 1046/1077] MANTIS 0060518 CLA - Check if PLF sets
 OneIdAppTransfer interface version "1.1.0" explicit version: use default
 instead of fixed

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1653 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/idl/Cuss/CUSS-APP-Transfer-OneID.xsd             | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/CUSS-APP-Transfer-OneID.xsd b/SelfServiceCommon/Massai/idl/Cuss/CUSS-APP-Transfer-OneID.xsd
index 4ada5acb..ad1f25b5 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/CUSS-APP-Transfer-OneID.xsd
+++ b/SelfServiceCommon/Massai/idl/Cuss/CUSS-APP-Transfer-OneID.xsd
@@ -7,17 +7,17 @@
 			<xs:documentation>schema version; increase fixed values of version when schema is changed</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="majorVersion" type="xs:int" fixed="1">
+			<xs:element name="majorVersion" type="xs:int" default="1">
 				<xs:annotation>
 					<xs:documentation>major version</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="minorVersion" type="xs:int" fixed="1">
+			<xs:element name="minorVersion" type="xs:int" default="1">
 				<xs:annotation>
 					<xs:documentation>minor version</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element name="revision" type="xs:int" fixed="0">
+			<xs:element name="revision" type="xs:int" default="0">
 				<xs:annotation>
 					<xs:documentation>revision</xs:documentation>
 				</xs:annotation>
-- 
2.41.0.windows.1

